src/s1reader/s1_annotation.py

```diff
@@ -897,3 +912,120 @@ def _anx2height(cls, delta_anx):
             h_t += h_i * np.sin((i+1) * worb * delta_anx + phi[i])
 
         return h_t
+
+
+@dataclass
+class BurstExtendedCoeffs:
+    '''
+    Segments of FM rate / Doppler centroid polynomial coefficients.
+    For (linear) interpolation of FM rate / Doppler Centroid along azimuth.
+    To be used for calculating azimuth FM rate mismatch mitigation
+    '''
+
+    # FM rate
+    vec_aztime_fm_rate: np.ndarray
+    lut_coeff_fm_rate: np.ndarray
+    vec_tau0_fm_rate:np.ndarray
+
+    # Doppler centroid
+    vec_aztime_dc: np.ndarray
+    lut_coeff_dc: np.ndarray
+    vec_tau0_dc: np.ndarray
+
+    @classmethod
+    def from_polynomial_lists(cls,
+                              az_fm_rate_list: list,
+                              doppler_centroid_list: list,
+                              sensing_start: datetime.datetime,
+                              sensing_end: datetime.datetime):
+        '''
+        Extract coefficients from `product_annotation` that fall within
```

Comment:
Looks like docstring needs to be revised (and I will). In the beginning of this PR, the lists of polynomials were stored in the data class `ProductAnnotation`, which was redundant because `s1_reader.burst_from_xml()` already loads the list of the polynomial coefficients.